src/server/control.rs

```diff
@@ -50,6 +50,7 @@ pub fn instance_command(cmd: &InstanceCommand) -> anyhow::Result<()> {
                 return status::print_status_all(c.extended, c.debug, c.json);
             }
         }
+        _ => panic!("Should not be here"),
```

Comment:
There is `unreachable!()` macro for exactly this case. But please add specific `Create` and `Destroy` cases here and mention where they were handled.

This is important that when a new command is added compiler highlights all the matches where it should be added.